File: app/src/main/java/com/nikhil/grapqldemo/ui/users/UsersFragment.java
package com.nikhil.grapqldemo.ui.users;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import androidx.lifecycle.ViewModelProvider;

import com.nikhil.grapqldemo.R;
import com.nikhil.grapqldemo.databinding.FragmentUsersBinding;
import com.nikhil.grapqldemo.ui.base.BaseFragment;


public class UsersFragment extends BaseFragment {

    private FragmentUsersBinding binding;
    private UsersViewModel viewModel;

    public UsersFragment() {
        // Required empty public constructor
    }

    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container,
                             Bundle savedInstanceState) {
        binding = customizeDataBinding(inflater, container, R.layout.fragment_users);
        return binding.getRoot();
    }

    @Override
    public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
        showToolbar();
        showBottomNavigation();
        hideBackButton();
        viewModel = new ViewModelProvider(this).get(UsersViewModel.class);
    }
}

File: app/src/main/java/com/nikhil/grapqldemo/ui/users/UsersFragment.kt
package com.nikhil.grapqldemo.ui.users

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.lifecycle.ViewModelProvider
import com.nikhil.grapqldemo.R
import com.nikhil.grapqldemo.databinding.FragmentUsersBinding
import com.nikhil.grapqldemo.ui.base.BaseFragment

class UsersFragment : BaseFragment() {

    private var binding: FragmentUsersBinding? = null
    private var viewModel: UsersViewModel? = null
    override fun onCreateView(
        inflater: LayoutInflater, container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View {
        binding = customizeDataBinding(inflater, container, R.layout.fragment_users)
        return binding!!.root
    }

    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
        showToolbar()
        showBottomNavigation()
        hideBackButton()
        viewModel = ViewModelProvider(this)[UsersViewModel::class.java]
    }
}